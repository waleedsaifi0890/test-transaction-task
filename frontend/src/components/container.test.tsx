import React from 'react';
import { expect, test } from 'vitest';
import renderer from 'react-test-renderer';
import Container from '@app/components/container';

test('snapshot', () => {
  const result = renderer
    .create(
      <Container>
        <p>Hello World</p>
      </Container>
    )
    .toJSON();

  expect(result).toMatchSnapshot();
});
