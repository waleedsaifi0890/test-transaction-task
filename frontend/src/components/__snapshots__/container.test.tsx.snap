// Vitest Snapshot v1

exports[`snapshot 1`] = `
<div
  className="container"
>
  <p>
    Hello World
  </p>
</div>
`;
